sdcm/scan_operation_thread.py

```diff
@@ -383,7 +383,9 @@ def _compare_output_files(self) -> bool:
                 self.log.debug("Compared output of normal and reversed queries is identical!")
                 result = True
             else:
-                self.log.warning("Normal and reversed queries output differs: \n%s", result.stdout.strip())
+                output = result.stdout.strip()
+                self.log.warning("Normal and reversed queries output differs (diff-size is %s): \n%s",
```

Comment:
Maybe should be an error?
No one looks at warning...

Reply:
@roydahan , the problem is we cannot tell it's a real error for sure.
It can be confirmed only if there's no background load or any deletion nemesis running.
That's why this check is less functional and more statistic, in the overall results.
(Perhaps, optimally, we would save the failures/success statistics to ES or something)

Reply:
I disagree. any delete nemesis or additional writes aren't or at least shouldn't happen for this validated range.
This range is only being read during the test.

Reply:
also no one would be looking on error prints, this should be an event with error level, and fail the test.
otherwise it's pointless verification

Reply:
@roydahan , @fruch , i think what was defined as the main goal for this validation is challenging scylla's reversed queries in terms of sanity to see no coredumps / memory issues / other exceptions, during load + deletions + cache + multiple sstables/ memtables updates.
The data validation is quite a bonus with this regard.
Now what you are suggesting above is a much limited and quite "static" validation. It mean validating data that was only written once during prepare.
So a solution could be (when `validate-date: true`) to validate data only if the partition is in range of `partition_range_with_data_validation` . This will have a dependency and requires https://github.com/scylladb/scylla-cluster-tests/pull/6185 to be merged first. please confirm.

Reply:
```
self.log.warning("Normal and reversed queries output differs (diff-size is %s lines): \n%s <...clipped...>",
```